frontend/index.html

```diff
@@ -605,14 +604,22 @@ <h2 class="contact">Contact</h2>
               }
               $("#cList").append("<li>" + RESEARCH_QUESTION_TEXT + "</li>");
             }
-          });
         });
       })};
 
       //Page load order
 
-      //Get data
-      getLang();
+      const STUDY_NAME = _getStudyName(window.location);
+      async function fetchText() {
+        const response = await fetch(`https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`);
+        const textFetch = await response.json();
+        return textFetch;
+      }
+
+      fetchText().then(textFetch => { 
+        window.text = textFetch;
```

Comment:
why do you need to set window.text here?
Given that you also do `const data = window.text;`, why can't you just directly do

```
const data = textFetch;
```

Reply:
This was done because of the scope of the variable, since the variable that stores the JSON must be read from a different function than the one that performs the fetch. Therefore, a "global variable" is created so that it can be read from the function that modifies the text in the HTML.

Reply:
good point that you do need to put it into a global variable. But I don't see why that global variable needs to be hung onto the `window`. You have tons of other global variables (`curr_lang` for example).

Reply:
I understand but I have tried all methods, but only the `window` function seems to work.